### PR TITLE
NF-818 - updated Welsh translations

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -248,23 +266,24 @@ emailAddress.error.required = Nodwch gyfeiriad e-bost
...
 repaymentType.01 = Diwygiad mis cyfredol (CMA)
 repaymentType.02 = Trosglwyddiad banc (Bacs)
+repaymentType.hint.01 = Dim ond os gwnaethoch y taliad gwreiddiol gan ddefnyddio cyfrif gohirio tollau y gallwch ddefnyddio’r opsiwn hwn.
 repaymentType.checkYourAnswersLabel = Dull talu
 repaymentType.error.required = Dewiswch sut yr hoffech gael yr ad-daliad
 
 enterAgentEORI.title = Beth yw rhif EORI y mewnforiwr?
 enterAgentEORI.heading = Beth yw rhif EORI y mewnforiwr?
-enterAgentEORI.hint = Mae hyn yn 2 lythyren, wedi’u dilyn gan naill ai 12 neu 15 rhif, fel ‘GB123456789123’ neu ‘GB123456789123000’.
+enterAgentEORI.hint = Mae hwn yn cynnwys 2 lythyren, wedi’u dilyn gan naill ai 12 neu 15 rhif. Er enghraifft, ‘GB123456789123’ neu ‘GB123456789123000’.
```
Comment (njmoorewitchcat) [inline conf/messages.cy:286]: AMBER: We have removed the single quotes around the example in English.  Have the Welsh translation team accidentally added them back in?

File: translations/NDRC Welsh Translation_20210810.csv [hunk 2]
```diff
@@ -1,728 +1,1521 @@
...
+"whomToPay.error.required"	"Select who you want HMRC to repay"	"Dewiswch pwy yr hoffech i CThEM ad-dalu"	Added.
+"furtherInformation.hintText"	"This may be information that HMRC has asked for, information you choose to add or a request to cancel your application."	"Gall hyn fod yn wybodaeth y mae CThEM wedi gofyn amdani, gwybodaeth y byddwch yn dewis ei hychwanegu neu gais i ganslo’ch cais."	Added.
+"entryDetails.importer.hint"	"You can find these on the E2 or C88 documents. If you’re unsure, you can speak to your shipping agent or courier."	"Gallwch ddod o hyd i’r rhain ar y dogfennau E2 neu C88. Os ydych yn ansicr, gallwch siarad â’ch asiant cludo neu’ch cludwr."	Added.
+"entryDetails.representative.hint"	"You can find these on the E2 or C88 documents."	"Gallwch ddod o hyd i’r rhain ar y dogfennau E2 neu C88."	Added.
+"enterAgentEORI.hint"	"This has 2 letters, followed by either 12 or 15 numbers. For example, GB123456789123 or GB123456789123000."	"Mae hwn yn cynnwys 2 lythyren, wedi’u dilyn gan naill ai 12 neu 15 rhif. Er enghraifft, ‘GB123456789123’ neu ‘GB123456789123000’."	Added.
```
Comment (njmoorewitchcat) [inline translations/NDRC Welsh Translation_20210810.csv:763]: AMBER: Seems like the Welsh translation team have added quotes around these example, where the English does not have quotes around them (on purpose).

**Review by njmoorewitchcat — DISMISSED** [review]
Highlighted the single quote addition by the Welsh translation team.  I believe we removed them based on our content designers decision. Not going to reject this, if the Welsh team say add quotes, then we add quotes!

File: translations/NDRC Welsh Translation_20210810.csv (hunk 2 above)
Comment (SteveSugden) [reply translations/NDRC Welsh Translation_20210810.csv:763]: Removed the single quotes from the "source" csv file and re-ran the migration tools

**Review by njmoorewitchcat — APPROVED** [review]

**Pull request merged**